RELEASING.md

```diff
@@ -0,0 +1,39 @@
+- [Overview](#overview)
+- [Branching](#branching)
+  - [Release Branching](#release-branching)
+  - [Feature Branches](#feature-branches)
+- [Releasing](#releasing)
+- [Snapshots](#snapshot-builds)
+
+## Overview
+
+This document explains the release strategy for artifacts in this organization.
+
+## Branching
+
+### Release Branching
+
+Given the current major release of 0.1, projects in this organization maintain the following active branches.
```

Comment:
This document is generic to all upcoming and future releases. `0.1` is more of a pre-release than actual major release which will be `1.0`

Reply:
I can change it to `1.0`.